Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM governmentpaas/cf-cli
+
+WORKDIR /usr/src/app
+
+COPY Gemfile Gemfile.lock .ruby-version ./
+
+RUN apk --update --upgrade add build-base openssl && \
```

Comment:
I have a working version of this locally with a slightly terser syntax:

```Dockerfile
RUN apk --update add g++ musl-dev make nodejs
RUN bundle install
```

I don't think we need to delete the cache or the packages afterwards, as this image will just be thrown away? 

(Not sure how much it's worth messing around with this either)